README.md

```diff
@@ -51,7 +51,9 @@ To learn more about {gsm}'s data pipeline, visit the [Data Pipeline Vignette](ht
 Detailed RMarkdown/HTML reporting is built into `{gsm}`, and provides a detailed overview of all risk assessments for a given trial.
 
 Reports for a sample trial run with [`{clindata}`](https://github.com/Gilead-BioStats/clindata) are provided below:
+
 - [Site Report](https://gilead-biostats.github.io/gsm/reports/StandardReportSite.html)
```

Comment:
these links (site and country) are giving 404 errors, when I try and use them, it will work with https://gilead-biostats.github.io/gsm but not when you add /reports or /StandardReportSite.html, not sure if this is just a place holder for now?

Reply:
Good catch! I should have noted this in the PR - this will be resolved once the `pkgdown` site re-deploys after the release.